### PR TITLE
button-widget.c: replace tabs with 4 spaces

### PR DESCRIPTION
**Pull request by lukefromdc**
*and reformat as necessary

### CODE REVIEW

**Comment by lukefromdc** [comment]
One possible way of doing this, I am open to other options as well. Objective is easier to read and maintain code.
This is a reformat of white space only. Should be NO changes to built binary and NO change to anything but a tab, a space, or in a few cases a blank line or an opening bracket position.

A test built and run gave no surprises and identical runtime behavior in both x11 and Wayland, ANY difference would have indicated a typo somewhere. We need to make sure no typos get in with this sort of reformat, note that GNOME did this with an automated tool some years ago, using a 2 spaces instead of tabs format. Most new code in MATE we use four spaces so that's what I used here.  Original tabs appear to have been intended for 8 spaces but little further work was needed to line everything up with 4 spaces, which saves space and matches most of our newer work.

**Comment by lukefromdc** [comment]
@raveit65 , if you want to do this yourself in a different way, feel free to either force-push to this branch, or open another PR and I will close this one in  preference to it as you are probably our best formatter. Had some time so decided this might save you some.

**Comment by raveit65** [comment]
Cool, i will take a look at it later and push changes for myself to this branch.

**Comment by raveit65** [comment]
Seems like the file was a mix of tab-size=4/8. o_0 No wonder that a fine formatting was impossible.
Using spaces everywhere and 4 spaces for indents will terminate this confusion.

**Comment by raveit65** [comment]
I'd like to take another look and do a runtime test before review and merging.

File: mate-panel/button-widget.c [hunk 1]
```diff
@@ -57,876 +57,873 @@ G_DEFINE_TYPE_WITH_PRIVATE (ButtonWidget, button_widget, GTK_TYPE_BUTTON)
...
 
-	if (button->priv->surface_hc)
-		cairo_surface_destroy (button->priv->surface_hc);
-	button->priv->surface_hc = NULL;
+    button->priv->surface = NULL;
+
+    if (button->priv->surface_hc)
+        cairo_surface_destroy (button->priv->surface_hc);
+
+    button->priv->surface_hc = NULL;
```
Comment (raveit65) [inline mate-panel/button-widget.c:158]: @mate-desktop/core-team 
Not sure about adding a blank line after the if clauses.
`button->priv->surface = NULL;`
and
`button->priv->surface_hc = NULL;`
doesn't belongs to the if clauses?

**Comment by lukefromdc** [comment]
Those are single-line if clauses w no brackets, thus the blank lines beneath

File: mate-panel/button-widget.c (hunk 1 above)
Comment (raveit65) [reply mate-panel/button-widget.c:158]: good than adding a blank line is OK.

**Comment by raveit65** [comment]
I tested that change with x11 and wayland. Everything works as expected.
Please review.
Commits can be squashed in one during merge.

**Comment by lukefromdc** [comment]
I am on the road now away from my desktop, will look it over when I get home. Will also build and install, testing in both x11 and wayland

**Review by lukefromdc — COMMENTED** [review]
This is really nice work you added, and with the overall reformatting the build warnings we used to get about misleading indents at the `if` statements are gone

File: mate-panel/button-widget.c (hunk 1 above)
Comment (cwendling) [reply mate-panel/button-widget.c:158]: If indentation is correct either way is fine, but I don't think a blank line is a bad idea, so I'm happy with it.

**Pull request merged**